src/sentry/static/sentry/app/data/experimentConfig.tsx

```diff
@@ -31,6 +31,12 @@ export const experimentList = [
     parameter: 'variant',
     assignments: ['1', '0', -1],
   },
+  {
+    key: 'OnboardingSidebarV2Experiment',
+    type: 'organization',
+    parameter: 'variant',
```

Comment:
Shouldn't this be 'exposed'?
https://github.com/getsentry/getsentry/blob/master/getsentry/planout/onboarding_sidebar_v2_experiment.py

Reply:
Yes it should thank you